openwisp_controller/connection/connectors/ssh.py

```diff
@@ -163,15 +163,14 @@ def exec_command(
         # log standard error
         error = stderr.read().decode('utf-8', 'ignore')
         if error:
-            logger.error(error)
```

Comment:
The error is already getting appended to the output in following lines. We should not log it here.